meta-resin-common/recipes-containers/docker-disk/files/entry.sh

```diff
@@ -10,7 +10,7 @@ PARTITION_SIZE=${PARTITION_SIZE:=1024}
 
 # Create a BTRFS disk image
 dd if=/dev/zero of=/export/data_disk.img bs=1M count=$PARTITION_SIZE
-mkfs.btrfs /export/data_disk.img
+mkfs.btrfs --mixed --metadata=single /export/data_disk.img
```

Comment:
I reckon @petrosagg is more competent to answer this question. Personally I did not do much investigation on the matter.